### PR TITLE
[SHACK-358] update docs and navigation for chef.sh

### PR DESCRIPTION
**Pull request by marcparadise**
### Description

This restructures the CLI reference documents to be under the
heading "Tools Reference" and adds placeholders for key chefdk tools.

Previously, the "CLI Reference" heading was under "Chef Workstation" and
contained only the chef-run CLI reference. This change puts Workstation
tool CLI reference docs under a single heading to make them easier to
find.

The reference doc for chef-run is complete, but the other components currently
contain links over the their docs.chef.io (or inspec.io/docs)
counterparts until the docs migration effort is complete. 
capture
capture

### Check List

- [x] PR title is a worthy inclusion in the CHANGELOG
- [x] You have locally validated the change
- [x] `www/site/content/docs/` has been updated with any relevant changes:
  - * new or changed error messages in 'troubleshooting.md'
  - * new or changed CLI flags in cli-reference.md

### CODE REVIEW

File: www/site/content/docs/tools-reference/berks.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
++++
+title = "Tools Reference - berks"
```
Comment (cheeseplus) [inline www/site/content/docs/tools-reference/berks.md]: Seems overkill to have `Tools Reference` on each page when that will be clearly indicated by the Nav

Comment (marcparadise) [reply www/site/content/docs/tools-reference/berks.md]: I went back and forth on that - what I actually wanted was to get the full name in the <head> title, "Chef Workstation Tools Reference - TOOLNAME", while the menu item and the page heading just have the tool name. 
I'll push the update to be just the tool name, and can open another card to straighten out titles on the browser titlebar (which will be only  TOOLNAME with that change), vs what's shown on the menu and in the content content.

Comment (marcparadise) [reply www/site/content/docs/tools-reference/berks.md]: Updated, new screencaps inbound

**Comment by kagarmoe** [comment]
I'd like to just call it "Reference", in keeping with our other docs sites.

**Comment by marcparadise** [comment]
@kagarmoe thanks, I've updated and refreshed the screen captures.

**Comment by jonsmorrow** [comment]
@kagarmoe I'd like to get your sign off before we merge this.

**Pull request merged**